src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/UpdateJson.cs

```diff
@@ -23,21 +23,25 @@ public class UpdateJson : Task
         [Required]
         public string PathToAttribute { get; set; }
```

Comment:
We may want to just use S.T.J at some point instead of NewtonSoft.Json?

Reply:
There is a tracking issue for this - https://github.com/dotnet/source-build/issues/921